libraries/shared/factories/storage/transformer_test.go

```diff
@@ -140,11 +132,11 @@ var _ = Describe("Storage transformer", func() {
 				ID:       rand.Int63(),
 				HeaderID: fakeHeaderID,
 				RawDiff: types.RawDiff{
-					HashedAddress: common.Hash{},
-					BlockHash:     common.HexToHash(fakeBlockHash),
-					BlockHeight:   fakeBlockNumber,
-					StorageKey:    common.Hash{},
-					StorageValue:  rawValue.Hash(),
+					Address:      fakes.FakeAddress,
```

Comment:
nbd at all but could be cool to use `test_data.FakeAddress()` here - hoping to eventually remove `fakes.FakeAddress` (since it's the same every time) and always replace with something that's randomly generated on demand